eng/pipelines/templates/jobs/ci.yml

```diff
@@ -67,16 +67,25 @@ jobs:
           ServiceDirectory: ${{ parameters.ServiceDirectory }}
           TestPipeline: ${{ parameters.TestPipeline }}
 
-  - ${{ if ne(parameters.RunUnitTests, false) }}:
-    - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
-      parameters:
-        JobTemplatePath: /eng/pipelines/templates/jobs/ci.tests.yml
-        MatrixConfigs: ${{ parameters.MatrixConfigs }}
-        MatrixFilters: ${{ parameters.MatrixFilters }}
-        MatrixReplace: ${{ parameters.MatrixReplace }}
-        CloudConfig:
-          Cloud: Public
-        AdditionalParameters:
+  - job: Compliance
+    pool:
+      name: azsdk-pool-mms-win-2019-general
+      vmImage: MMS2019
+    steps:
+      - template: /eng/common/pipelines/templates/steps/credscan.yml
+        parameters:
           ServiceDirectory: ${{ parameters.ServiceDirectory }}
-          Artifacts: ${{ parameters.Artifacts }}
-          TestProxy: ${{ parameters.TestProxy }}
+
+  - ${{ if ne(parameters.RunUnitTests, false) }}:
+      - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
```

Comment:
Yes not only does the indention look off it looks like this is in the incorrect spot. It should be above the Compliance job.

Reply:
Actually nevermind about the ordering as this is an entire job template so it can go above or below the Compliance job. However do double check the indention.

Reply:
It is working as expected and indentation is correct.